PWGJE/EMCALJetTasks/macros/TrackEfficiencyConfiguration.yaml

```diff
@@ -15,7 +15,12 @@ LHC18r:
     0_10: [0.963, 0.963 , 0.940 , 0.936 , 0.949 , 0.958 , 0.977 , 0.967 , 0.976 , 0.966 , 0.975 , 0.975 , 0.975 , 0.982 , 0.982, 0.982] 
     30_50: [0.965, 0.965 , 0.958 , 0.956 , 0.959 , 0.967 , 0.986 , 0.978 , 0.986 , 0.978 , 0.986 , 0.986 , 0.986 , 0.986 , 0.986 , 0.986]
 
-# p-Pb - production don't have anchored period info so we use the MC tag
+# p-Pb 
+LHC16q:
+    0_100: [0.989 , 0.989 , 0.981 , 0.988 , 0.986 , 0.982 , 0.978 , 0.978 , 0.978 , 0.974 , 0.974 , 0.974 , 0.974 , 0.974 , 0.974 , 0.974]
+LHC16t:
+    0_100: [0.989 , 0.989 , 0.981 , 0.988 , 0.986 , 0.982 , 0.978 , 0.978 , 0.978 , 0.974 , 0.974 , 0.974 , 0.974 , 0.974 , 0.974 , 0.974]
```

Comment:
If we have to keep copying due to different period name's with the same performance, we can try an approach to reduce this via yaml anchors (basically pointers). eg:

```yaml
LHC16t: &LHC16t_anchor_name
    0_100: [0.989 , 0.989 , 0.981 , 0.988 , 0.986 , 0.982 , 0.978 , 0.978 , 0.978 , 0.974 , 0.974 , 0.974 , 0.974 , 0.974 , 0.974 , 0.974]
LHC17h6e2:
    <<: *LHC16t_anchor_name
```